src/network/mcpe/protocol/types/event/PortalUsedEventData.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+declare(strict_types=1);
+
+namespace pocketmine\network\mcpe\protocol\types\event;
+
+use pocketmine\network\mcpe\protocol\EventPacket;
+use pocketmine\network\mcpe\serializer\NetworkBinaryStream;
+
+final class PortalUsedEventData implements EventData{
+	/** @var int */
+	public $toDim;
+	/** @var int */
+	public $dim;
```

Comment:
these field names are unclear.

Reply:
it doesn't make sense to rigorously adhere to these names. This is not a carbon copy implementation, it is merely designed to match the binary format.

Reply:
I would be happier if these were named `currentDimension` and `targetDimension` respectively.